### ISSUE TITLE
Entity load multiple needed?

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/views_data_export#11]
I was looking through a codebase for something else and happened to see [this line](https://github.com/backdrop-contrib/views_data_export/blob/1.x-3.x/views_data_export.module#L67):

```
foreach (entity_load('file', FALSE, array('uri' => $uri)) as $file) {
```

It looks like that is an outdated use of `entity_load` that should be changed to `entity_load_multiple`.

**Comment by herbdool** [comment]
Your PR works for me @laryn

**Comment by argiepiano** [comment]
This change is fine for clarity, but `entity_load()` in Backdrop actually invokes `entity_load_multiple()` if it finds that the second argument is an array or `FALSE` or `NULL`... so in the end the correct function is used either way.

**Comment by laryn** [comment]
Yes, I think I went on a little spree one day after `grep`ing my codebase and @argiepiano corrected me on another thread too. Oops!

**Comment by argiepiano** [comment]
To be honest I prefer `entity_load_multiple()`!
